os/hal/ports/SN32/LLD/SN32F2xx/USB/hal_usb_lld.h

```diff
@@ -390,7 +390,9 @@ struct USBDriver {
 #define usb_lld_wakeup_host(usbp)                                           \
   do {                                                                      \
     SN32_USB->SGCTL = (mskBUS_DRVEN|mskBUS_K_STATE);                        \
-    osalThreadSleepMilliseconds(SN32_USB_HOST_WAKEUP_DURATION);             \
+    uint32_t loops = OSAL_MS2I(SN32_USB_HOST_WAKEUP_DURATION) *             \
+                     (48000000 / CH_CFG_ST_FREQUENCY / 9);                  \
```

Comment:
it's probably better to use `SN32_HCLK` instead of hardcoding the value here

Reply:
Done now